klippy/extras/bed_mesh.py

```diff
@@ -402,9 +421,19 @@ def cmd_BED_MESH_MAP(self, params):
     def cmd_BED_MESH_CALIBRATE(self, params):
         self.build_map = False
         self.start_calibration(params)
+    cmd_BED_MESH_TILT_help = "Perform Tilting of existing Mesh"
+    def cmd_BED_MESH_TILT(self, params):
+        self.start_tilting(params)
     def start_calibration(self, params):
         self.bedmesh.set_mesh(None)
+        self.probed_z_table_backup=None
         self.probe_helper.start_probe(params)
+    def start_tilting(self, params):
+        if self.bedmesh.z_mesh.mesh_z_table is None:
```

Comment:
When the mesh is clear `self.bedmesh.z_mesh` is set to None.  The check above will raise an exception.  It is sufficent for the check to be:
```
if self.bedmesh_zmesh is None:
```

Reply:
Indeed :)